zokrates_core/src/flatten/mod.rs

```diff
@@ -1082,38 +1080,24 @@ impl<'ast, T: Field> Flattener<'ast, T> {
         }
     }
 
-    fn flatten_u_to_bits(
+    fn u_to_bits(
```

Comment:
I left these functions in to avoid code duplication here https://github.com/Zokrates/ZoKrates/pull/957/commits/ed97e815c71f61e88cb484fb33fb2563cdf4b8a5#diff-c3ae48a36bfbdcad0b4a5d7861eea7bcbb116730f80c3832e77012b4988f248dR1131